### DISCUSSION TITLE
Love this

### DISCUSSION THREAD

**Discussion by Brleegb11** [TT-ReBORN/Georgia-ReBORN#162] [General]
Only downloaded yesterday - love it is the best music player I ever seen easy!

**Comment by giorgiobrit** [comment]
Easy! It's Gorgeous! I hope the spider monkey will upgrade to 64bit and next update include the tree library function. Happy new year Tom and friends!

**Comment by TT-ReBORN** [comment]
Glad you like it!

all I can say is 2024 will be a good year for Georgia-ReBORN!

-Tom
